articles/stream-analytics/cosmos-db-managed-identity.md

```diff
@@ -43,23 +43,13 @@ For the Stream Analytics job to access your Cosmos DB using managed identity, th
 |---------|
 |Cosmos DB Built-in Data Contributor|
 
-1. Select **Access control (IAM)**.
+Today, CosmosDB does not support adding RBAC roles through Azure Portal, so Azure Powershell is required to be able to assign the correct permissions to authenticate your ASA job with your CosmosDB account. Azure Powershell can be accessed through Azure portal. 
```

Comment:
Technically, cosmos db allows you to add RBAC roles via portal. But i think the specific role ASA job needs (to auth with MSI) is not available on the portal. I think we still need to callout what the actual role name is apart from the sample role definition id which is ambiguous.

Reply:
We should explicitly mention that Cosmos DB data plane built-in role-based access control (RBAC) are not exposed through the Azure Portal. 

But I do see that the role name is mentioned here Cosmos DB Built-in Data Contributor